config/locales/doc_auth/en.yml

```diff
@@ -70,6 +70,14 @@ en:
           flash on your camera is off and try taking a new picture.
         top_msg_plural: We couldn’t read your ID. Your photos may have glare. Make sure
           that the flash on your camera is off and try taking new pictures.
+      http:
+        image_load: There is something wrong with your image file. Please take new
+          photos of your ID and try again.
+        image_size: You image size is too large or too small. Please add images of your
+          ID that are about 2025 x 1275.
```

Comment:
should we specify units (pixels) here like the other languages do?

Reply:
huh, how'd that get missed? Good catch. Thanks!